openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -680,20 +680,21 @@
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-245
+        Red Hat OpenShift enables auditing by default for the API Servers,
+        the OAuth server and the Linux hosts themselves. Auditing cannot
+        be turned off which makes the control inherently met.
```

Comment:
should we move the implementation status to complete?

Reply:
sure, done